### DISCUSSION TITLE
Getting started with editing default pages

### DISCUSSION THREAD

**Discussion by fvlasie** [breezefront/community#118] [1. Help] [answered]
Hello everyone,

I am new to Breeze themes and I have just installed Breeze Evolution for testing.

I would like to know if there is a tutorial on editing the default pages. I tried to edit the Evlolution home page in page builder but it does not always behave the way I expect. Is there a way to edit the source code for the defualt pages directly? I could not find the code in app/design/frontend where I expected it to be. Is there a different process for working with Breeze themes? 

Thank you!

**Comment by vovayatsyuk** [comment, marked as answer]
CMS pages are stored in database, there are no "files" for them, so you have to use Magento pagebuilder. Magento gives you the ability to disable pagebuilder via configuration and write the pages/blocks/description using HTML code, but in this case, you will have to write your styles for all pages.

**Reply by fvlasie** [reply]
Thank you!
